frontend/src/config.ts

```diff
@@ -7,6 +7,8 @@ export const HOST_API = process.env.REACT_APP_HOST_API_KEY || '';
 
 export const MAPBOX_API = process.env.REACT_APP_MAPBOX_API;
 
+export const isDevelopment = !process.env.NODE_ENV || process.env.NODE_ENV === 'development';
```

Comment:
nit: !process.env.NODE_ENV is a useless condition?

Since if process.env.NODE_ENV = undefined, it won't equals to development anyway.

Reply:
quoted from: https://stackoverflow.com/questions/35469836/detecting-production-vs-development-react-at-runtime

> This is best done emulating the Node way of doing things with your build tool - webpack, browserify - by exposing process.env.NODE_ENV. Typically, you'll have it set to "production" in prod and "development" (or undefined) in dev.

Reply:
@chownces do we need to set this env var to 'production' in prod ?